### PR TITLE
fill in rospkg rosdep keys

### PR DESCRIPTION
**Pull request by tfoote**
Adds modules and python3 support

### CODE REVIEW

File: rosdep/python.yaml [hunk 1]
```diff
@@ -3501,30 +3501,34 @@ python-rospkg:
...
-    yakkety: [python-rospkg]
     yakkety_python3: [python3-rospkg]
-    zesty: [python-rospkg]
     zesty_python3: [python3-rospkg]
+python-rospkg-modules:
+  alpine:
+    pip:
+      packages: [rospkg]
+  arch: [python2-rospkg]
+  debian: [python-rospkg-modules]
```
Comment (dirk-thomas) [inline rosdep/python.yaml]: I doubt this is a valid Debian package name?

Same below.

Comment (tfoote) [reply rosdep/python.yaml]: Why not? We package it for debian ourselves.

From: http://packages.ros.org/ros-shadow-fixed/ubuntu/dists/stretch/main/binary-amd64/Packages
```
Package: python-rospkg-modules
Priority: optional
Section: python
Installed-Size: 130
Maintainer: Ken Conley <kwc@willowgarage.com>
Architecture: all
Version: 1.1.9-1
Replaces: python-rospkg (<< 1.1.0)
Depends: python-yaml, python:any (<< 2.8), python:any (>= 2.7.5-5~), python-catkin-pkg-modules, lsb-release
Conflicts: python-rospkg (<< 1.1.0)
Filename: pool/main/p/python-rospkg-modules/python-rospkg-modules_1.1.9-1_all.deb
SHA512: b6230a36265fa24c0f7d01ad2caadef2a2b32c9ea1a11364a164552b477d550d595cb76bad4654aa2610c774a482c0f190c75e3ef78622ecc71d2d1db1007a5d
Size: 23680
SHA256: 808c1eaf8266bef7039e0660b36d63f7143fc7872635c05b627b4c2f20b209ac
SHA1: 35ae5feec927adfff7ee9462bbbd73345ca2ced1
MD5sum: 54762cb2612805ab190dbc2c0e04f265
Description: ROS package library
         Library for retrieving information about ROS packages and stacks.
Homepage: http://wiki.ros.org/rospkg
```

Comment (dirk-thomas) [reply rosdep/python.yaml]: Doesn't the Debian rule apply to vanilla Debian too?

Comment (tfoote) [reply rosdep/python.yaml]: Everything in this database assumes that you have the ROS sources available so ours packaged version will take precedence.

Comment (dirk-thomas) [reply rosdep/python.yaml]: It seems to be inconsistent with other `-modules` packages though: https://github.com/ros/rosdistro/blob/24ea867b61062bff1056961905f2c88210940bf0/rosdep/python.yaml#L4759

**Comment by dirk-thomas** [comment]
Can you mention where the new rosdep keys are planned to be used (https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#rosdep-rules-contributions).

**Comment by tfoote** [comment]
rospkg's modules are being used from a python3 script in this package to find resources: https://github.com/osrf/drone_demo/blob/xacro_models/sitl_launcher/package.xml

**Review by nuclearsandwich — APPROVED** [review]

**Review by dirk-thomas — APPROVED** [review]

**Pull request merged**